proofs/cbmc/poly_decompose/Makefile

```diff
@@ -26,6 +26,7 @@ USE_DYNAMIC_FRAMES=1
 # Disable any setting of EXTERNAL_SAT_SOLVER, and choose SMT backend instead
 EXTERNAL_SAT_SOLVER=
 CBMCFLAGS=--smt2
+CBMC_FLAG_UNSIGNED_OVERFLOW_CHECK=
```

Comment:
@rod-chapman - do you know if still should still be needed?

Reply:
Aha! Well spotted - let me restore the original and re-run...

Reply:
It's fine with the original flags, which is good news. Rebased, squashed, and pushed.